POKEMON_OF_THE_DAY/src/clients/pokemon/index.ts

```diff
@@ -0,0 +1,66 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from "axios";
+import { Pokemon, PokemonType, PokemonTypeSearchResult } from "./models";
+export default class Client {
+  _axiosInstance: AxiosInstance;
+  constructor() {
+    this._axiosInstance = axios.create({
+      baseURL: `https://pokeapi.co/api/v2/`,
+    });
+  }
+
+  getPokemon = async (pokemonId: string) =>
```

Comment:
the current formatting may be intentional for beginner understanding ... but if not I think this could be something like to be a bit more typescript-y
```
getPokemon = async (pokemonId: string) =>
                 this.getResource<Pokemon>("pokemon", pokemonId);
```
and 
```
async getResource<T>(resource: string, id?: number | string): Promise<T>
```